src/main/javascript/Runner.js

```diff
@@ -0,0 +1,754 @@
+            isAt(x - 1, y, element) ||
+            isAt(x, y + 1, element) ||
+            isAt(x, y - 1, element);
+    };
+
+    var countNear = function (x, y, element) {
+        if (pt(x, y).isOutOf(size)) {
+            return 0;
+        }
+        var count = 0;
+        if (isAt(x - 1, y, element)) count++; // TODO to remove duplicate
+        if (isAt(x + 1, y, element)) count++;
+        if (isAt(x, y - 1, element)) count++;
+        if (isAt(x, y + 1, element)) count++;
+        return count;
+    };
+
+    return {
+        size: boardSize,
+        getMe: getMe,
+        getOtherHeroes: getOtherHeroes,
+        isGameOver: isGameOver,
+        isAt: isAt,
+        isAtMany: isAtMany,
+        getAt: getAt,
+        boardAsString: boardAsString,
+        getBarriers: getBarriers,
+        toString: toString,
+        findAll: findAll,
+        getAccelerators: getAccelerators,
+        getFences: getFences,
+        getInhibitors: getInhibitors,
+        getLineUpChangers: getLineUpChangers,
+        getLineDownChangers: getLineDownChangers,
+        getObstacles: getObstacles,
+        getSpringboardDarkElements: getSpringboardDarkElements,
+        getSpringboardLightElements: getSpringboardLightElements,
+        getSpringboardLeftDownElements: getSpringboardLeftDownElements,
+        getSpringboardRightDownElements: getSpringboardRightDownElements,
+        getSpringboardLeftUpElements: getSpringboardLeftUpElements,
+        getSpringboardRightUpElements: getSpringboardRightUpElements,
+        getSpringboardTopElements: getSpringboardTopElements,
+        checkNearMe: checkNearMe,
+        checkNearMeManyMoves: checkNearMeManyMoves,
+        isOutOfFieldRelativeToMe: isOutOfFieldRelativeToMe,
+        hasFenceAt: hasFenceAt,
+        hasInhibitorAt: hasInhibitorAt,
+        hasAcceleratorAt: hasAcceleratorAt,
+        hasObstacleAt: hasObstacleAt,
+        hasUpLineChangerAt: hasUpLineChangerAt,
+        hasDownLineChangerAt: hasDownLineChangerAt,
+        hasSpringboardDarkElementAt: hasSpringboardDarkElementAt,
+        hasSpringboardLightElementAt: hasSpringboardLightElementAt,
+        hasSpringboardLeftDownElementAt: hasSpringboardLeftDownElementAt,
+        hasSpringboardRightDownElementAt: hasSpringboardRightDownElementAt,
+        hasSpringboardTopElementAt: hasSpringboardTopElementAt,
+        hasOtherBikeAt: hasOtherBikeAt,
+        isAnyOfAt: isAnyOfAt,
+        isNear: isNear,
+        isBarrierAt: isBarrierAt,
+        countNear: countNear
+    };
+};
+
+var random = function (n) {
+    return Math.floor(Math.random() * n);
+};
+
+var direction;
+
+var DirectionSolver = function (board) {
+    return {
+        /**
+         * @return next hero action
+         */
+        get: function () {
+            var me = board.getMe();
+            //console.log(me.getX(), me.getY());
+
+            // TODO your code here
+            var dir = Direction.STOP;  // STUB get any random direction except Direction.STOP
```

Comment:
Could we accent here that only two Direction options are supported by ExciteBike?

Reply:
sure